.github/workflows/discord-notifications.yml

```
@@ -0,0 +1,30 @@
+            ${{ secrets.DISCORD_WEBHOOK_URL }}
+
+      - name: PR Merged Notification
+        if: github.event.action == 'closed' && github.event.pull_request.merged == true
+        run: |
+          curl -H "Content-Type: application/json" \
+            -d "{\"embeds\": [{\"title\": \"✅ PR Merged\", \"description\": \"**${{ github.event.pull_request.title }}**\", \"url\": \"${{ github.event.pull_request.html_url }}\", \"color\": 5763719, \"fields\": [{\"name\": \"Author\", \"value\": \"${{ github.event.pull_request.user.login }}\", \"inline\": true}, {\"name\": \"Merged by\", \"value\": \"${{ github.event.sender.login }}\", \"inline\": true}]}]}" \
+            ${{ secrets.DISCORD_WEBHOOK_URL }}
+
+      - name: PR Closed Without Merge Notification
+        if: github.event.action == 'closed' && github.event.pull_request.merged == false
+        run: |
```

actionlint (1.7.9)

"github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)